### PR TITLE
Refactor and promote ecosystem and tooling doc

### PR DESCRIPTION
**Pull request by alexwolfmsft**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/dotnet-ai-ecosystem.md](https://github.com/dotnet/docs/blob/8f05774b5d03f872133b943c84fd458117ad6a33/docs/ai/dotnet-ai-ecosystem.md) | [docs/ai/dotnet-ai-ecosystem](https://review.learn.microsoft.com/en-us/dotnet/ai/dotnet-ai-ecosystem?branch=pr-en-us-41944) |
| [docs/ai/semantic-kernel-dotnet-overview.md](https://github.com/dotnet/docs/blob/8f05774b5d03f872133b943c84fd458117ad6a33/docs/ai/semantic-kernel-dotnet-overview.md) | [docs/ai/semantic-kernel-dotnet-overview](https://review.learn.microsoft.com/en-us/dotnet/ai/semantic-kernel-dotnet-overview?branch=pr-en-us-41944) |

### CODE REVIEW

File: docs/ai/dotnet-ai-ecosystem.md [hunk 1]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
 
-Today, you can use .NET to access models built by OpenAI, using either the Azure OpenAI SDK or the Semantic Kernel. These models can be hosted by OpenAI or in Azure using the Azure AI services. Preview support is coming soon in Semantic Kernel to work with other models, and you can experiment today using open-source SDKs created by the .NET developer community.
+Visit the [Semantic Kernel documentation](/semantic-kernel/overview/) for more information.
 
-| NuGet package | Supported models | Maintainer or vendor | Link to docs |
+## .NET SDKs for building AI apps
+
+Many different SDKs are available for .NET to build apps with AI capabilities depending on the target platform or AI model. OpenAI models offer powerful generative AI capabilities, while other Azure AI Services provide intelligent solutions for a variety of specific scenarios.
+
+### .NET SDKs for OpenAI Models
```
Comment (jmatthiesen) [inline docs/ai/dotnet-ai-ecosystem.md:30]: For now, the strongest story is around OpenAI models, but there will be others like Mistral that we want to promote. I think this section is fine for now, but we should also add the official OpenAI package: https://www.nuget.org/packages/OpenAI/

File: docs/ai/dotnet-ai-ecosystem.md [hunk 2]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+| [Azure AI Language](/azure/ai-services/language-service/) | Build apps with industry-leading natural language understanding capabilities. |
+| [Azure AI Speech](/azure/ai-services/speech-service/) | Speech to text, text to speech, translation, and speaker recognition. |
+| [Azure AI Translator](/azure/ai-services/translator/) | AI-powered translation technology with support for more than 100 languages and dialects. |
+| [Azure AI Vision](/azure/ai-services/computer-vision/) | Analyze content in images and videos. |
+
+## Develop with local AI models
+
+.NET apps can also connect to local AI models for many different development scenarios. [Semantic Kernel](https://github.com/microsoft/semantic-kernel) is the recommended tool to connect to local models using .NET. Semantic Kernel can connect to many different models hosted across a variety of platforms and abstracts away lower level implementation details.
+
+For example, you can use [Ollama](https://ollama.com/) to run various AI models locally, including several Small Language Models (Slms) developed by Microsoft:
```
Comment (jmatthiesen) [inline docs/ai/dotnet-ai-ecosystem.md:56]: We should add a link to the Local Models quickstart, to help demonstrate how to use these.

File: docs/ai/includes/vector-databases.md [hunk 3]
```diff
@@ -22,3 +24,5 @@ AI applications often use data vector databases and services to improve relevanc
 - [Redis](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Redis)
 - [Sqlite](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Sqlite)
 - [Weaviate](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Weaviate)
```
Comment (jmatthiesen) [inline docs/ai/includes/vector-databases.md:26]: We should also highlight Milvus, Qdrant, Pinecone, and Postgres in this list.

File: docs/ai/includes/vector-databases.md [hunk 4]
```diff
@@ -5,7 +5,9 @@ ms.date: 07/29/2024
 ms.topic: include
 ---
 
-AI applications often use data vector databases and services to improve relevancy and provide customized functionality. Many of these services provide a native SDK for .NET, which you can use directly. Semantic Kernel provides an extensible component model that enables you to use different vector stores without needing to learn each SDK. Semantic Kernel provides connectors for the following vector databases and services:
+AI applications often use data vector databases and services to improve relevancy and provide customized functionality. Many of these services provide a native SDK for .NET, while others offer a REST service you can connect to through custom code. Semantic Kernel provides an extensible component model that enables you to use different vector stores without needing to learn each SDK.
+
+Semantic Kernel provides connectors for the following vector databases and services:
 
 - [Vector Database in Azure Cosmos DB for NoSQL](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL)
```
Comment (jmatthiesen) [inline docs/ai/includes/vector-databases.md:12]: Suggest alphabetizing the list as well. The Cosmos DB scenarios probably aren't the most interesting to our customers, so they also don't make sense at the top of the list.

File: docs/ai/includes/vector-databases.md (hunk 3 above)
Comment (jmatthiesen) [inline docs/ai/includes/vector-databases.md:26]: We should still promote 3rd party SDKs for vector DBs, which can be used without SK. Maintaining these from the prior page. While we are promoting SK, we still want to show there's an ecosystem outside of it.

| [Azure.Search.Documents](https://www.nuget.org/packages/Azure.Search.Documents/) | Azure AI Search | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft) | [Azure AI Search client library for .NET](/dotnet/api/overview/azure/search.documents-readme) |
| [Milvus.Client](https://www.nuget.org/packages/Milvus.Client) | Milvus Vector Database | [Milvus](https://milvus.io/) | [Install Milvus C# SDK](https://milvus.io/docs/v2.2.x/install-csharp.md) |
| [Qdrant.Client](https://www.nuget.org/packages/Qdrant.Client) | Qdrant Vector Database | [Qdrant](https://qdrant.tech) | [Qdrant .NET SDK](https://github.com/qdrant/qdrant-dotnet) |

**Review by jmatthiesen — CHANGES_REQUESTED** [review]

File: docs/ai/includes/vector-databases.md (hunk 3 above)
Comment (alexwolfmsft) [reply docs/ai/includes/vector-databases.md:26]: These are on the list, but they're hidden in the PR diff.

File: docs/ai/dotnet-ai-ecosystem.md [hunk 5]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+
+### .NET SDKs for OpenAI Models
+
+| NuGet package | Supported models | Maintainer or vendor | Documentation |
 |---------------|------------------|----------------------|--------------|
 | [Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/) | [OpenAI models](https://platform.openai.com/docs/models/overview)<br/>[Azure OpenAI supported models](/azure/ai-services/openai/concepts/models) | [Semantic Kernel](https://github.com/microsoft/semantic-kernel) (Microsoft) | [Semantic Kernel documentation](/semantic-kernel/) |
 | [Azure OpenAI SDK](https://www.nuget.org/packages/Azure.AI.OpenAI/) | [Azure OpenAI supported models](/azure/ai-services/openai/concepts/models) | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft) | [Azure OpenAI services documentation](/azure/ai-services/openai/) |
+| [OpenAI SDK](https://www.nuget.org/packages/OpenAI/) | [Azure OpenAI supported models](https://platform.openai.com/docs/models) | [OpenAI SDK for .NET](https://github.com/openai/openai-dotnet) (OpenAI) | [OpenAI services documentation](https://platform.openai.com/docs/overview) |
```
Comment (jmatthiesen) [inline docs/ai/dotnet-ai-ecosystem.md:36]:
```suggestion
| [OpenAI SDK](https://www.nuget.org/packages/OpenAI/) | [OpenAI supported models](https://platform.openai.com/docs/models) | [OpenAI SDK for .NET](https://github.com/openai/openai-dotnet) (OpenAI) | [OpenAI services documentation](https://platform.openai.com/docs/overview) |
```

Fixed a typo

**Review by jmatthiesen — APPROVED** [review]

File: docs/ai/dotnet-ai-ecosystem.md [hunk 6]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
-
-> [!NOTE]
-> We recommend using the [Semantic Kernel SDK](/semantic-kernel/overview/) to orchestrate your calls to large language models (LLMs) and manage interactions with the various services mentioned here. Semantic Kernel makes it easy to work with different services without having to learn a different API for each one.
+The .NET ecosystem provides many powerful tools, libraries, and services to develop AI applications. .NET supports both cloud and local AI model connections, many different SDKs for various AI and vector database services, and other tools to help you build intelligent apps of varying scope and complexity.
 
 > [!IMPORTANT]
-> These SDKs and tools are built by a variety of sources. Not all SDKs are maintained by Microsoft. When considering an SDK, be sure to evaluate its quality, licensing, and support to ensure they meet your requirements. Also make sure you review each SDK's documentation for detailed version compatibility information.
+> Not all of the SDKs and services presented in this doc are maintained by Microsoft. When considering an SDK, make sure to evaluate its quality, licensing, support and compatibility to ensure they meet your requirements.
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:14]:
```suggestion
> Not all of the SDKs and services presented in this doc are maintained by Microsoft. When considering an SDK, make sure to evaluate its quality, licensing, support, and compatibility to ensure they meet your requirements.
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 7]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+
+## Semantic Kernel for .NET
+
+[Semantic Kernel](semantic-kernel-dotnet-overview.md) is an open-source SDK that enables AI integration and orchestration capabilities in your .NET apps. This SDK is generally the recommended AI orchestration tool for .NET apps that use one or more AI services in combination with other APIs or web services, data stores, custom code and beyond. Semantic Kernel benefits enterprise developers in the following ways:
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:18]:
```suggestion
[Semantic Kernel](semantic-kernel-dotnet-overview.md) is an open-source SDK that enables AI integration and orchestration capabilities in your .NET apps. This SDK is generally the recommended AI orchestration tool for .NET apps that use one or more AI services in combination with other APIs or web services, data stores, and custom code. Semantic Kernel benefits enterprise developers in the following ways:
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 8]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
 
-## Working with models
+- Streamlines integration of AI capabilities into existing applications to enable a cohesive solution for enterprise products.
+- Minimizes the learning curve of working with different AI models or services by providing abstractions that reduce complexity.
+- Improves reliability by reducing the unpredictable behavior of prompts and responses from AI models. You can fine-tune prompts and plan tasks to create a controlled and predictable user experience.
 
-Today, you can use .NET to access models built by OpenAI, using either the Azure OpenAI SDK or the Semantic Kernel. These models can be hosted by OpenAI or in Azure using the Azure AI services. Preview support is coming soon in Semantic Kernel to work with other models, and you can experiment today using open-source SDKs created by the .NET developer community.
+Visit the [Semantic Kernel documentation](/semantic-kernel/overview/) for more information.
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:24]:
```suggestion
For more information, see the [Semantic Kernel documentation](/semantic-kernel/overview/).
```

File: docs/ai/dotnet-ai-ecosystem.md (hunk 1 above)
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:30]:
```suggestion
### .NET SDKs for OpenAI models
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 9]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
 | [Azure OpenAI SDK](https://www.nuget.org/packages/Azure.AI.OpenAI/) | [Azure OpenAI supported models](/azure/ai-services/openai/concepts/models) | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft) | [Azure OpenAI services documentation](/azure/ai-services/openai/) |
+| [OpenAI SDK](https://www.nuget.org/packages/OpenAI/) | [OpenAI supported models](https://platform.openai.com/docs/models) | [OpenAI SDK for .NET](https://github.com/openai/openai-dotnet) (OpenAI) | [OpenAI services documentation](https://platform.openai.com/docs/overview) |
+
+### .NET SDKs for Azure AI Services
+
+Azure offers many other AI services to build specific application capabilities and workflows. Most of these services provide a .NET SDK to integrate their functionality into custom apps. Visit the [Azure AI Services](/azure/ai-services/what-are-ai-services) documentation for a complete list of available services and learning resources. Some of the most commonly used services include the following:
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:40]:
```suggestion
Azure offers many other AI services to build specific application capabilities and workflows. Most of these services provide a .NET SDK to integrate their functionality into custom apps. Some of the most commonly used services are shown in the following table. For a complete list of available services and learning resources, see the [Azure AI Services](/azure/ai-services/what-are-ai-services) documentation.
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 10]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+| [Azure AI Content Safety](/azure/ai-services/content-safety/) | Detect unwanted or offensive content. |
+| [Azure AI Document Intelligence](/azure/ai-services/document-intelligence/) | Turn documents into intelligent data-driven solutions. |
+| [Azure AI Language](/azure/ai-services/language-service/) | Build apps with industry-leading natural language understanding capabilities. |
+| [Azure AI Speech](/azure/ai-services/speech-service/) | Speech to text, text to speech, translation, and speaker recognition. |
+| [Azure AI Translator](/azure/ai-services/translator/) | AI-powered translation technology with support for more than 100 languages and dialects. |
+| [Azure AI Vision](/azure/ai-services/computer-vision/) | Analyze content in images and videos. |
+
+## Develop with local AI models
+
+.NET apps can also connect to local AI models for many different development scenarios. [Semantic Kernel](https://github.com/microsoft/semantic-kernel) is the recommended tool to connect to local models using .NET. Semantic Kernel can connect to many different models hosted across a variety of platforms and abstracts away lower level implementation details.
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:54]:
```suggestion
.NET apps can also connect to local AI models for many different development scenarios. [Semantic Kernel](https://github.com/microsoft/semantic-kernel) is the recommended tool to connect to local models using .NET. Semantic Kernel can connect to many different models hosted across a variety of platforms and abstracts away lower-level implementation details.
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 11]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+
+For example, you can use [Ollama](https://ollama.com/) to [connect to local AI models with .NET](quickstarts/quickstart-local-ai.md), including several Small Language Models (Slms) developed by Microsoft:
+
+| Model | Description |
+| --- | --- |
+| [phi3 models](https://azure.microsoft.com/products/phi-3) | A family of powerful, small language models (SLMs) with groundbreaking performance at low cost and low latency. |
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:60]:
```suggestion
| [phi3 models](https://azure.microsoft.com/products/phi-3) | A family of powerful small language models (SLMs) with groundbreaking performance at low cost and low latency. |
```

File: docs/ai/dotnet-ai-ecosystem.md [hunk 12]
```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
...
+| [orca models](https://www.microsoft.com/en-us/research/project/orca/) | Research models in tasks such as reasoning over user given data, reading comprehension, math problem solving and text summarization. |
```
Comment (gewarren) [inline docs/ai/dotnet-ai-ecosystem.md:61]:
```suggestion
| [orca models](https://www.microsoft.com/en-us/research/project/orca/) | Research models in tasks such as reasoning over user given data, reading comprehension, math problem solving, and text summarization. |
```

**Review by gewarren — APPROVED** [review]
LGTM with a few suggestions for consideration.

**Pull request merged**